package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@demos-europe/demosplan-ui",
-  "version": "0.0.2",
+  "name": "my-webpack-project",
+  "version": "1.0.0",
   "license": "MIT",
   "private": false,
-  "description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan.",
+  "description": "My webpack project",
```

Comment:
I am pretty sure that solething like this is missing to make it usable in projects:
```
  "main": "./dist/demosplan-ui.umd.js",
  "module": "./dist/demosplan-ui.es.js",
  "exports": {
    ".": {
      "import": "./dist/demosplan-ui.es.js",
      "require": "./dist/demosplan-ui.umd.js"
    }
  },
  ```
  (at least "main")